miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -89,9 +97,16 @@ abstract class MiniAppMessageBridge {
             ActionType.REQUEST_PERMISSION.action -> onRequestPermission(callbackObj)
             ActionType.REQUEST_CUSTOM_PERMISSIONS.action -> onRequestCustomPermissions(jsonStr)
             ActionType.SHARE_INFO.action -> onShareContent(callbackObj.id, jsonStr)
+            ActionType.LOAD_AD.action -> onLoadAd(callbackObj.id, jsonStr)
+            ActionType.SHOW_AD.action -> onShowAd(callbackObj.id, jsonStr)
         }
     }
 
+    /** Set implemented ads displayer. **/
+    fun setAdMobDisplayer(miniAppAdDisplayer: MiniAppAdDisplayer) {
```

Comment:
It's used only in `MiniAppMessageBridgeSpec` 🤔

Reply:
The hostapps can use it when they don't want to use default implementation. It's better to be declared in document later 👍

Reply:
I see, in that sense, probably we can define `AdMobDisplayer` as public if HostApp wants a ready-made `MiniAppAdDisplayer`.

Reply:
It could be a good idea.
However, my idea is let the hostapp implement and pass the `MiniAppAdDisplayer` interface instead of `AdMobDisplayer` class.
What do you think @khairul-alam-licon @corycaywood ?

Reply:
Understood, set `AdMobDisplayer` as public and the default should return error to miniapp.